client/server/config/parser.js

```diff
@@ -57,15 +57,6 @@ module.exports = function ( configPath, defaultOpts ) {
 		} );
 	}
 
-	if (
-		secretsPath !== realSecretsPath &&
-		data.features &&
-		data.features[ 'wpcom-user-bootstrap' ]
-	) {
-		console.error( 'Disabling server-side user-bootstrapping because of a missing secrets.json' );
-		data.features[ 'wpcom-user-bootstrap' ] = false;
-	}
-
```

Comment:
This would be nice to keep in some form, because without it calypso.live with the `wpcalypso` env will not work. The `wpcom-user-bootstrap` flag is enabled in that env, but calypso.live has no `secrets.json` and all requests will be failing and will be [redirecting to login](https://github.com/Automattic/wp-calypso/blob/trunk/client/server/pages/index.js#L330-L352).

Instead of checking for `secrets.json` presence, we should be checking if the `wpcom_calypso_*_api_key` were set or not, from whatever source.

Reply:
Fixed.